client/src/components/judging/JudgingRoom.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import roomData = require('../../data.json');
+
+
+interface JudgingRoomProps {
+    table: string;
+}
+
+// let styles = {
+//     fill:none;stroke:#000000;stroke-width:3.14576221px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1
+// }
+
+class JudgingRoom extends React.Component<JudgingRoomProps, {}>{
+    constructor(props: JudgingRoomProps) {
+        super(props)
+    }
+    public render() {
+        let index = 0;
+        const tableColor = this.props.table.split(" ")[0].toLowerCase();
+        if (tableColor === "green") {
+            index = 1;
+        } else if (tableColor === "orange") {
+            index = 2;
+        }
+        let tableId = 0
```

Comment:
be consistent with whether you use semicolons or not;

Reply:
Use semicolons